### ISSUE TITLE
[Bug]: Firefox Won't Open WACZ from Remote Server because the Size of the File is Not Accessible

### ISSUE THREAD

**Issue by markpbaggett** [webrecorder/replayweb.page#318]
### ReplayWeb.page Version

v2.0.0

### What did you expect to happen? What happened instead?

When trying to replay a WACZ in Firefox, I get this error message:

```
Sorry, this URL could not be loaded because the size of the file is not accessible.
Make sure this is a valid URL and you have access to this file.
```

Interestingly, this works fine in both Chrome and Safari.  I'm not sure if this is related to how I captured the WACZ or something else.

In case it's helpful, I used Browsertrix-Crawler and this command:

```
docker run -v $PWD/crawls:/crawls/ -it webrecorder/browsertrix-crawler crawl --url https://abolition-now.github.io/an --generateWACZ --text --collection abolition_now_test
```

This opens fine in replayweb.page locally, but won't open in Firefox remotely:

```
https://digital.lib.utk.edu/demo/abolition_now_test.wacz
```

Is the web server not returning a header or something that replayweb.page is expecting?

### Step-by-step reproduction instructions

1.  Open Firefox
2. Navigate to replayweb.page
3. Attempt to open this externally hosted resource:  https://digital.lib.utk.edu/demo/abolition_now_test.wacz

**Comment by Shrinks99** [comment]
> Is the web server not returning a header or something that replayweb.page is expecting?

Yep!  @edsu recently wrote a [good forum post](https://forum.webrecorder.net/t/sorry-this-url-could-not-be-loaded-because-the-size-of-the-file-is-not-accessible/557/7?u=hank) on exactly this problem for another user experiencing the same thing.  `Content-Length` is missing and is required for WACZ playback.

**Comment by ikreymer** [comment]
The issue is unfortunately due to a bug/incorrect handling in Firefox. Chrome and Safari correct add `Accept-Encoding: identity` (the default) header when a range request is sent, allowing the server to return content-length. Unfortunately, Firefox always sends `Accept-Encoding: gzip, deflate, br, zstd`, which causes the header to return a compressed version of the WACZ, which is not what we want.

Barring the fix in Firefox, I think the best option is to ensure your server ignores the Accept-Encoding header for WACZ files.


</body></html>
